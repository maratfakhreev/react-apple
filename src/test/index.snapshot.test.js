import React from 'react';
import renderer from 'react-test-renderer';
import Apple from '../apple';

describe('Apple', () => {
  it('renders correctly', () => {
    const params = {
      className: 'iPhone',
      'data-key': 'value'
    };
    const appleComponent = renderer.create(
      <Apple { ...params }>
        <h1>Test</h1>
      </Apple>
    ).toJSON();

    expect(appleComponent).toMatchSnapshot();
  });
});
