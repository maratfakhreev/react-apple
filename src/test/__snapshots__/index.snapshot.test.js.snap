// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Apple renders correctly 1`] = `
<apple
  className="apple-iPhone"
  data-key="value"
>
  <h1>
    Test
  </h1>
</apple>
`;
